common/answers/10-defaults.sh

```diff
@@ -3,7 +3,6 @@
 # SET SOME SANE DEFAULTS
 {
   echo "RACHER_SERVER=${RANCHER_SERVER-rancher.nhschoices.net}"
```

Comment:
I'm not sure setting RACHER_SERVER  default is still needed. I could be wrong tho'!

Reply:
@neilbmclaughlin you're right, something to remove at some point in the future.